modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/StructuredContentResourceImpl.java

```diff
@@ -700,6 +698,24 @@ private String _createJournalArticleContent(
 		}
 	}
 
+	private UnsafeConsumer<BooleanQuery, Exception>
+		_createStructuredContentPageUnsafeConsumer(Boolean flatten) {
```

Comment:
Same as above. Might be _createStructuredContentsPageBooleanQueryUnsafeConsumer?